.github/workflows/python_type_checks.yml

```diff
@@ -0,0 +1,34 @@
+name: Python type checks
+on:
+  push:
+    branches:
+      - develop
+      - release-*
+  pull_request:
+    branches:
+      - develop
+      - release-*
+
+jobs:
+  run_tests:
+    name: Run Mypy Type Checks
+    runs-on:  ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '2.x'
+          architecture: 'x64'
+      - name: Setup python by installing wheel
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        run: pip install wheel==0.35.0
+      - uses: ./.github/actions/merge
+      - name: Install Third Party Dependencies
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        run: python -m scripts.install_third_party_libs
```

Comment:
Do we still need this step?

Reply:
`install_third_party_libs.main()` inside `run_mypy_checks` would install all the packages and is that why you are saying this line becomes redundant @DubeySandeep?

We also import python utils in `run_mypy_checks` and it might raise some external library import error perhaps? Not sure about that. But I see this pattern is only being used in other checkers also like `run_backend_tests` - see [here](https://github.com/oppia/oppia/blob/140c2919b6a8ba5e28c701558bf0fa2efc76d485/.github/workflows/backend_tests.yml#L29) and [here](https://github.com/oppia/oppia/blob/140c2919b6a8ba5e28c701558bf0fa2efc76d485/scripts/run_backend_tests.py#L69)? Do we plan to fix those too?

Reply:
> We also import python utils in run_mypy_checks and it might raise some external library import error perhaps

Yes the import errors were thrown without the install_third_party_libs in the workflow.

Reply:
We had that pattern in circleCI for some reason to avoid install libs multiple times, but I think now we can remove those lines!

Reply:
@DubeySandeep I tried running checks without install_third_party_libs, errors were thrown because of missing imports.

Reply:
@DubeySandeep this will be handled by @hardikkat24 using --skip-install flag as mentioned in #13088 in his upcoming PR #13114.